cassandra/src/main/scala/io/zipkin/dependencies/spark/cassandra/ZipkinDependenciesJob.scala

```diff
@@ -137,25 +123,48 @@ case class ZipkinDependenciesJob(sparkMaster: String = ZipkinDependenciesJob.spa
   }
 
   def saveToCassandra(sc: SparkContext, keyspace: String, dependencies: DependenciesInfo): Unit = {
-    val thrift = dependencies.toDependencies(startTs = startTs, endTs = endTs).toThrift
-    val blob: Array[Byte] = Codecs.dependenciesCodec.encode(thrift).array()
+    val thrift = Dependencies.create(startTs,  endTs, dependencies.links.asJava)
+    val blob: Array[Byte] = thrift.toThrift.array()
 
-    val day = Time.fromMilliseconds(endTs).floor(Duration.fromTimeUnit(1, DAYS)).inMilliseconds
```

Comment:
bug! endTs is often midnight, so this was storing spans in the daily bucket *after* their timestamps.